### ISSUE TITLE
New Website

### ISSUE THREAD

**Issue by gamesarefunlol** [HasangerGames/suroi#219]
Hi,

You should add suroi2.io. Suroi.io has gotten blocked at my school and it’s annoying. Please do this. I love this game.

**Comment by hsanger** [comment]
Try https://pr.suroi.io

**Comment by ChpssCode** [comment]
That doesn’t work either.

**Comment by gamesarefunlol** [comment]
Haha lol. I’m still gonna play games. But that site still doesn’t work,On Dec 5, 2023, at 5:37 PM, Damien Vesper ***@***.***> wrote:﻿
Skill issue. Stop playing games in class and focus on your schoolwork.

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>

(This transmittal and/or attachments have been sent from a
student of the North Allegheny School District)

**Comment by Katloo24** [comment]
You’re welcome I just deleted that last message, you doxxed your school district @gamesarefunlol

**Comment by CroissantEdit** [comment]
my school district is ausd

**Comment by kenos1** [comment]
> Skill issue. Stop playing games in class and focus on your schoolwork.

@DamienVesper what if you finish really early and have nothing to do

**Comment by CroissantEdit** [comment]
> > Skill issue. Stop playing games in class and focus on your schoolwork.
> 
> @DamienVesper what if you finish really early and have nothing to do

yeah @DamienVesper what if your just so good that you play games while doing schoolwork :trollface:

**Comment by CroissantEdit** [comment]
yeah @DamienVesper what if im just so good and did my hw using chatgpt

**Comment by DamienVesper** [comment]
> yeah @DamienVesper what if im just so good and did my hw using chatgpt

You have no speaking rights.

**Comment by CroissantEdit** [comment]
I have full A 😼

**Comment by leia-uwu** [comment]
> I have full A 😼

![image](https://github.com/HasangerGames/suroi/assets/69596237/3b89de44-8d46-4819-8080-5061283e2240)

**Comment by DamienVesper** [comment]
> I have full A 😼

only in physical noneducation

**Comment by CroissantEdit** [comment]
No I swear

**Comment by CroissantEdit** [comment]
@DamienVesper 
![image](https://github.com/HasangerGames/suroi/assets/145735670/33ee14cb-141b-454e-84c7-b3f3c7b380bb)

**Comment by CroissantEdit** [comment]
@DamienVesper jk this is my real grade 
![image](https://github.com/HasangerGames/suroi/assets/145735670/e809302d-beb0-43f7-8d13-42426bc57be4)
i hate english 💀
